portfolio/src/main/webapp/woof.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+    <h1> Woof. </h1>
+	<img src="/images/pet/IMG-0057.JPG" alt="Beef jerky loving dog" height="500">
```

Comment:
super nit: usually we keep presentation logic like "height" in CSS: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns
but you haven't got up to that part yet, so maybe you can put a HTML comment above this line to say "TODO: Move height to CSS" or something.

Reply:
Oh okay sounds good